en/docs/learn/customizing-automated-emails.md

```diff
@@ -54,6 +54,26 @@ template.
             `{{user.claim.homeAddress}}`.
     
         Before sending the email, WSO2 Identity Server will retrieve the user details from the user store configured for this tenant and replace the placeholders with the corresponding values of the user.
+        
+        !!! note
+            When you are using an identity claim as a placeholder in the **Account Conifrmation** template
+            for User Self Registration, you need to use the notation ```{{claimName}}``` instead of using 
+            ```{{user.claim.identity.claimName}}```. And additionally you need to send the respsective value 
+            for the placeholder in the API request payload, under the ```properties``` section. 
+            See the example request payload below.
```

Comment:
A sample request payload is given below.

Reply:
Resolved in https://github.com/wso2/docs-is/pull/1595/commits/939800515438ce4205d520b92622c07321f8bda5